src/wp-admin/includes/class-custom-image-header.php

```diff
@@ -1314,12 +1314,23 @@ final public function get_header_dimensions( $dimensions ) {
 	 * Creates an attachment 'object'.
 	 *
 	 * @since 3.9.0
+	 * @deprecated 6.5.0
 	 *
 	 * @param string $cropped              Cropped image URL.
 	 * @param int    $parent_attachment_id Attachment ID of parent image.
 	 * @return array An array with attachment object data.
 	 */
 	final public function create_attachment_object( $cropped, $parent_attachment_id ) {
+		_doing_it_wrong(
+			'custom_image_header_create_attachment_object',
+			sprintf(
+			// translators: 1: The Custom_Image_Header method that is no longer used by core. 2: The new function that replaces it.
+				__( 'The %1$s method has been replaced with %2$s.' ),
+				'Custom_Image_Header::create_attachment_object',
+				'wp_copy_parent_attachment_properties'
+			),
+			'6.5.0'
+		);
```

Comment:
We have dedicated helpers for that :)

```suggestion
		_deprecated_function( __METHOD__, '6.5.0', 'wp_copy_parent_attachment_properties()' );
```

Reply:
Huh. I did not know that existed. So much easier. Thanks!